src/tests/JIT/Regression/JitBlue/GitHub_57065/Runtime_57065.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+
+// Generated by Fuzzlyn v1.2 on 2021-08-06 19:31:41
+// Run on .NET 6.0.0-dev on Arm Linux
+// Seed: 5500224797583134883
```

Comment:
Did you hand modify this program?
And will it be generated using this Seed value?

Reply:
Yes, all fuzzling repos are hand modified because they usually don't return 100 on success.